app/controllers/alert_controller.rb

```diff
@@ -68,5 +70,14 @@ def set_session_data
     session[:alert_lastaction] = @lastaction
   end
 
+  def breadcrumbs_options
+    {
+      :breadcrumbs => [
+        {:title => _("Cloud Intel")},
+        {:title => _("RSS")},
+      ],
```

Comment:
There is [RFE](https://bugzilla.redhat.com/show_bug.cgi?id=1672933) for removing RSS from UI. So there is no need to have it in breadcrumbs

Reply:
Deleted! :dagger: